### DISCUSSION TITLE
Change Font Size of Axis Labels / Tick Labels

### DISCUSSION THREAD

**Discussion by sdawilson** [team-moeller/better-access-charts#105] [Q&A] [answered]
I've really enjoyed seeing this development... excellent work.

I have a problem I just can't seem to solve... How do you change the font size of axis labels. Everything I've tried has failed. Any help very gratefully received.

**Comment by team-moeller** [comment]
BetterAccessCharts does not support the feature you are looking for.
I currently don't have the time for it. If you like, you can look how the Font class is used elsewhere. This way you could expand the code base and ideally come back with a pull request.

**Comment by sdawilson** [comment, marked as answer]
Hi Thomas,

Thank you very much form your reply.

I kind of guessed as much, but my knowledge of JS and Chart JS is next to negligible, so I’ll going to struggle to implement anything anytime soon.

A very impressive and elegant concept though – well done!

Best regards,

Simon Wilson

**Comment by team-moeller** [comment]
Hi Simon,
THX for the kind words. I think I will close this discussion.
Best Regards!
Thomas
